specification/metrics/quantityinpricingunit.md

```diff
@@ -0,0 +1,31 @@
+# Quantity in Pricing Unit
```

Comment:
As we've seen columns mature and close on a common prefixed grouping pattern, I would really like to see this and `QuantityInUsageUnit` be called `PricingQuantity` and `UsageQuantity` so they sit next to their respective unit columns. These will be very out-of-place if not sitting next to the column that explains what the number means.

Reply:
@ijurica  do we need to vote on this change or is it something you have already agreed with the maintainers?

Reply:
Will be amended following results of poll....

Reply:
As discussed in WG session ... moving to PricingQuantity for column name